### PR TITLE
#159082364 Write tests to the existing functionality

### PR DESCRIPTION
**Pull request by ssewilliam**
### What does this PR do?
- Adds tests to the existing functionality

### Description of Task to be completed?
- The application codebase currently has no tests. Please bring it up to sufficient coverage by writing tests for existing functionality.

### How should this be manually tested?
- ./manage.py test

### Any background context you want to provide?
- I Installed Django-nose, coverage and Django-coverage in the virtual environment
### What are the relevant pivotal tracker stories?
- [#159082364](https://www.pivotaltracker.com/story/show/159082364)
### Screenshots
screen shot 2018-07-25 at 11 24 55

### Questions:
- Are these tests sufficient enough?

### CODE REVIEW

**Comment by geofrocker** [comment]
The test coverage for models is still low

**Comment by geofrocker** [comment]
Resolve the merge conflicts

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[report]
+omit =
+    */site-packages/*
+    config.py
+    */tests/*
+    */migrations/*
```
Comment (geofrocker) [inline .coveragerc]: EOF(No new line at the end of the file) missing here

Comment (ssewilliam) [reply .coveragerc]: @geofrocker so this is not a python file, does it mean it also has to follow like python standards such as EOF?

**Pull request merged**